shared/constants/methods-tags.ts

```diff
@@ -6,15 +6,15 @@
  * network for the dapp. It could also be that a method expects the
  * globally selected network to match some value in the request params itself.
  */
-export const methodsRequiringNetworkSwitch = [
-  'eth_sendTransaction',
-  'eth_sendRawTransaction',
-  'wallet_switchEthereumChain',
-  'wallet_addEthereumChain',
-  'wallet_watchAsset',
-  'eth_signTypedData_v4',
-  'personal_sign',
-];
+export const methodsRequiringNetworkSwitch = Object.freeze([
+  'eth_sendTransaction' as const,
+  'eth_sendRawTransaction' as const,
+  'wallet_switchEthereumChain' as const,
```

Comment:
I believe this can be removed behind the feature flag now? I think the reason it was here was that we wanted to show the correct data on which network you were switching from?